research/conditional/utils/argparse.py

```diff
@@ -40,6 +40,7 @@ def introduce_parser_arguments(
     # CORE training hyperparameters, almost always specified in baseline configs
 
     parser.add_argument("--n_steps", type=int, required=True)
+    parser.add_argument("--n_training_tokens_in_billions", type=int, default=None)
```

Comment:
n_training_tokens_in_billions has to be float instead of int

Reply:
fixed